.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules/
-svg/output/
```

Comment:
Wondering why you removed `svg/output` and `build` directories from the git ignore?